teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -548,6 +549,19 @@ interface Change {
     val date: Date
 }
 
+interface ChangeFiles {
+    val count: Int
+    val files: List<ChangeFile>
+}
+
+interface ChangeFile {
+    val beforeRevision: String?
+    val afterRevision: String?
+    val changeType: String
```

Comment:
What values `changeType` may have? If it has fixed set of values, it's better to use `enum` here.